README.md

```diff
@@ -41,6 +41,10 @@ options are listed below,
 | forceReconnectionThreshold | Threshold of force gRPC reconnection if network issue is encountered | 1                        |
 | forceTLS                   | Force use TLS for gRPC                                               | false                    |
 | sslTrustCAPath             | SSL: Trusted CA Path                                                 |                          |
+| sslCertChainPath           | SSL: Cert Chain Path, BanyanDB server not support mTLS yet           |                          |
+| sslKeyPath                 | SSL: Cert Key Path, BanyanDB server not support mTLS yet             |                          |
```

Comment:
I prefer to remove them since the server side doesn't support mTLS.

Reply:
The first commit is copying server-side TLS-related options here. Didn't realize the client only needs trustCA.

Reply:
I fixed it by #90